### ISSUE TITLE
Mention the advantage of reduced training time when used in dataset intensive ML tasks

### ISSUE THREAD

**Issue by ParbatBhattarai** [Kapil-Devkota/BPN_CUDA#2]
We should probably mention the advantage of using GPUs on training time as done in the paper referenced below. 
And, also mention that the use of GPU will not improve the efficiency (rather decrease it in some cases) in the case of small datasets and nodes. 
So, since the use of BPN is done in various ML tasks which require massive amount of training dataset, parallelization of BPN is often always useful.

"OCR and on-line handwritten recognition are prime
target applications for GPUs. Training time is a major
bottleneck for handwriting recognition (1-3 weeks,
depending on the language). Printed OCR, being offline,
is limited by test time (currently at about 1000
character per second). In both case, GPUs bring a
much needed speedup."
Steinkrau, D., Simard, P.Y., Buck, I.: Using GPUs for machine learning algorithms.
In: ICDAR 2005: Proc. of the Eighth International Conference on Doc. Analysis
and Recognition, Washington, DC, USA, pp. 1115–1119. IEEE Comp. Society, Los
Alamitos (2005
